pyroute2/ipdb/interface.py

```diff
@@ -779,9 +779,18 @@ def error(x):
             #
             # One simple way to work that around is to remove
             # secondaries first.
-            rip = sorted(removed['ipaddr'],
-                         key=lambda x: self['ipaddr'][x]['flags'],
```

Comment:
The max value so far is 0x800 (IFA_F_STABLE_PRIVACY), so 0xffff looks safe. But, from the other side, IFA_FLAGS is uint32, so 0xffffffff would work here better…

Ok then.